documentation/asynDriver.html

```diff
@@ -5778,7 +5778,12 @@ <h3 id="drvAsynIPServerPort">
   <ul>
     <li>portName - The portName that is registered with asynManager.</li>
     <li>serverInfo - The Internet host name and port number to listen for connections
-      on (e.g. "localhost:4002" for TCP, "localhost:4002 UDP" for UDP).</li>
+      on (e.g. "localhost:4002" for TCP, "localhost:4002 UDP" for UDP), which will
+      bind only to the loopback interface on many machines. This allows to specify a
+      host name or ip address for multi-homed machines. If you want to specify <b>*any*</b>
+      interface, use an empty host <pre>":port"</pre> or the any address <pre>"0.0.0.0:port"</pre>
+      . The address has to be an existing interface on the host or if the name cannot
+      be resolved, it will use the any address like in the past.</li>
```

Comment:
I think this may break existing scripts which rely on localhost:4002 to not do any binding to an IP address.
See my other comment in the code.
If you really want to bind to 127.0.0.1:4002, specify "127.0.0.1:4002" instead of "localhost:4002"

Reply:
So should it test for a valid IP address and use it? Otherwise a name or invalid IP address would behave like before...

Reply:
May be I wasn't clear enough:
What happens if someone specifies
nonexistinghost:4002
I would expect asyn to fail loudly instead of silently falling back to INADDR_ANY.
This is debatable.

But at minimum I would like to see a warning in the IOC log